README.md

```diff
@@ -60,18 +60,32 @@ export class AppModule {
 }
 ```
 
-3. **Create a first AdminUser to log in with**
+3. **Add the provided `AdminUser` to your orm config:**
+
+```ts
+// ormconfig.js
+const AdminUser = require('nestjs-admin').AdminUserEntity
+
+module.exports = {
+  /* ... */,
+  entities: [__dirname + '/src/**/*.entity.{js,ts}', AdminUser],
+  /* ... */,
+}
+
+```
+
+4. **Create a first AdminUser to log in with**
 
 The DefaultAdminModule exposes an AdminUser entity, which has credentials that allow you to login to the admin interface.
 
 > There's no easy option to use your own user entity for now. If you have this requirement, open an issue so that we can help you.
 
 ```bash
-# create the AdminUser schema in database
+# If you don't have synchronise option turned on in your orm config, create the AdminUser schema in database
```

Comment:
Actually no need for this. Even if they have the `synchronize` option, they won't have it in prod so they need the migration